app/models/user.rb

```diff
@@ -3,7 +3,8 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable,
          :confirmable
 
-  validates :email , email: {mode: :strict}
+  validates :email, email: { mode: :strict }
+  validates :username, presence: true, length: { minimum: 4 , maximum: 16 }
```

Comment:
Pretty sure, ka mēs gribētu, ka `username` ir arī unikāls, savādāk kā pie logina zināsi kuru usri logot iekšā?

Reply:
A nu es padomaju ka logins bus tikai caur email un tad username tīri tam lai nebutu jārāda email par to kurs uztaisija post, bet ok doma good , ieliku ka unique